globally installed fido does not run anymore

Test are not failing. 

Any ideas why this does not work again? @cilice @hglattergotz?

```sh
~/Code/reactive/fido master 23s
❯ fido
module.js:339
    throw err;
    ^

Error: Cannot find module 'babel-runtime/helpers/interop-require-default'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/klechelt/.npm-global/lib/node_modules/@reactive/fido/lib/bin/cli.js:4:30)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:457:10)
```

--- a/package.json
+++ b/package.json
@@ -25,6 +25,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "babel-runtime": "^5.8.29",
     "chalk": "^1.1.1",
     "chrono-node": "^1.0.8",
     "dateformat": "^1.0.11",
@@ -41,7 +42,6 @@
     "babel": "^5.8.29",
     "babel-core": "^5.8.33",
     "babel-eslint": "^4.1.1",
-    "babel-runtime": "^5.8.29",
     "eslint": "^1.9.0",
     "eslint-config-airbnb": "1.0.2",
     "nock": "^3.3.1",

